coins.json

```diff
@@ -1206,6 +1206,32 @@
       "clientDocs": "https://docs.harmony.one/home/harmony-networks/harmony-network-overview/mainnet"
     }
   },
+  {
+    "id": "Oasis",
+    "name": "Oasis",
+    "coinId": 5786211,
+    "slip44": 474,
+    "symbol": "ROSE",
+    "decimals": 9,
+    "blockchain": "OasisNetwork",
+    "derivationPath": "m/44'/474'/0'/0/0",
+    "curve": "ed25519HD",
```

Comment:
is this path / curve compatible with others like Ledger or Trezor?

Reply:
Oasis uses ed25519+BIP32 derivation (not SLIP10 but Khovratovich style). This is also the default approach used by Ledger devices.
The Cardano implementation in Trustwallet seems to use something very similar, however, there are additional steps that make it incompatible with BIP39.
For this reason, we've created a new option. We know it is compatible with Ledger devices because we also developed the Ledger app for Oasis.

Reply:
good to know, make sure we have test case to cover this new added curve